config/sequelize.js

```diff
@@ -1,28 +1,28 @@
-import Sequelize from 'sequelize';
-import _ from 'lodash';
-import config from './config';
-
-const dbLogging = config.env !== 'test';
-
-const db = {};
-
-// // connect to postgres db
-const sequelize = new Sequelize(
-    config.postgres.db,
-    config.postgres.user,
-    config.postgres.passwd,
-    {
-        dialect: 'postgres',
-        port: config.postgres.port,
-        host: config.postgres.host,
-        logging: dbLogging,
-    }
-);
-
-db.User = sequelize.import('../server/models/user.model');
-
-// assign the sequelize variables to the db object and returning the db.
-module.exports = _.extend({
-    sequelize,
-    Sequelize,
-}, db);
+import Sequelize from 'sequelize';

```

Comment:
Are you using a different linting spec? Looks like nothing changed but github is seeing +/- on every line. Please make sure to grab the `.eslint` in the repo and run `yarn beautify` before each commit.